### PR TITLE
Pick a shorter name for the transfer ownership table

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Comment by nickvergessen** [comment]
Before installation failed on oracle:
https://travis-ci.org/nextcloud/notifications/jobs/621104031#L1359

File: apps/files/lib/Migration/Version11301Date20191205150729.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			'length' => 20,
+		]);
+		$table->addColumn('node_name', 'string', [
+			'notnull' => true,
+			'length' => 255,
+		]);
+		$table->setPrimaryKey(['id']);
+
+		// Quite radical, we just assume no one updates cross beta with a pending request.
+		// Do not try this at home
```
Comment (ChristophWurst) [inline apps/files/lib/Migration/Version11301Date20191205150729.php]: :see_no_evil:

File: apps/files/lib/Migration/Version11301Date20191113195931.php [hunk 2]
```diff
@@ -42,6 +42,8 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		/** @var ISchemaWrapper $schema */
 		$schema = $schemaClosure();
 
+		/*
+		 * Disabled because it breaks installing on Oracle
```
Comment (ChristophWurst) [inline apps/files/lib/Migration/Version11301Date20191113195931.php]: you could also drop this migrations. saves a useless intermediate table creation.

Comment (nickvergessen) [reply apps/files/lib/Migration/Version11301Date20191113195931.php]: it does not create a table anywhere? The full block is commented out?

Comment (ChristophWurst) [reply apps/files/lib/Migration/Version11301Date20191113195931.php]: sorry. I meant deleting the class if it doesn't execute anything …

Comment (rullzer) [reply apps/files/lib/Migration/Version11301Date20191113195931.php]: @nickvergessen ping

**Review by rullzer — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**